app/services/newsletter_events.rb

```diff
@@ -22,21 +29,21 @@ def self.create_campaign
     # Monday, 15th January 2018, 9.30am to 12.30pm
   end
 
-  def self.organisation_emails
+  def organisation_emails
```

Comment:
🤔 , why are the class methods converted to instance? (just my curiousity)

Reply:
hey @sigu how are you? we are using instance private methods in order to the methods to be visible only for the class, and for following the same pattern that is being used in the other services :+1: